syntax = "proto3";

package creport;

service Creport{
  // A simple RPC.
  //
  // Obtains the MessageResponse at a given position.
 rpc GetServerResponse(Message) returns (MessageResponse) {}

}

message Message{
 string message = 1;
}

message MessageResponse{
 string j_score = 1;
 string b_score = 2;
 string invest_money = 3;
 string sales = 4;
 bool received = 5;
}